src/Tribe/RSVP.php

```diff
@@ -918,8 +918,10 @@ public function save_ticket( $post_id, $ticket, $raw_data = array() ) {
 			return false;
 		}
 
-		// Updates if we should show Description
+		// Updates if we should show Type and Description
+		$ticket->show_type = isset( $ticket->show_type ) && tribe_is_truthy( $ticket->show_type ) ? 'yes' : 'no';
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 8 spaces but found 1 space

<sub>hash: 019c004a56c81ecc0455f99f3c6c4cbc</sub>

Reply:
Let's address this.

Reply:
```suggestion
		$ticket->show_type = ( isset( $ticket->show_type ) && tribe_is_truthy( $ticket->show_type ) ) ? 'yes' : 'no';
```